pulp_rpm/app/viewsets.py

```diff
@@ -115,15 +116,170 @@ class PackageViewSet(SingleArtifactContentUploadViewSet):
     minimal_serializer_class = MinimalPackageSerializer
     filterset_class = PackageFilter
 
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["create"],
+                "principal": "authenticated",
+                "effect": "allow",
```

Comment:
Are these separate statements for a reason?  (Probably they are, just asking a question)

Reply:
It is more about readability. Group ``list`` with ``create`` doesn't seem right.